bb_common/src/version.rs

```diff
@@ -56,6 +56,7 @@ block_version![
   [V1_17, V1_17_1],
   [V1_18, V1_18_2],
   [V1_19, V1_19_4],
+  [V1_20, V1_20],
```

Comment:
1.20.1 is already out, we might as well just jump to that.
```suggestion
  [V1_20, V1_20_1],
```

Reply:
I saw that you skipped same Protocol Version like 1.9.1 or 1.19.1. So i Skipped 1.20.1 because it has the same Version as 1.20

Reply:
sweet, sounds good. I'm going to hold off on merging until bamboo-data is updated, but this looks good to me.